File: android/ECellVSM/app/src/main/java/com/aasim/ecellvsm/Models/Completed.java
package com.aasim.ecellvsm.Models;

/**
 * Created by aasim on 26/12/16.
 */

public class Completed {
    String userid;
    String name;
    String longname ;
    float  quantity;
    float price;

    public String getUserid() {
        return userid;
    }

    public String getName() {
        return name;
    }

    public String getLongname() {
        return longname;
    }

    public int getQuantity() {
        return (int)quantity;
    }

    public float getPrice() {
        return price;
    }
}

File: node/models/completed.js
/**
 * Created by nidhi on 16/12/13.
 */
function completed(){
    this.userid = null;
    this.name = null;
    this.longname = null;
    this.quantity = null;
    this.price = null;
}

exports.create =  function () {
    var instance = new completed();
    return instance;
}